File: app/cs50x2024/content/french/lectures_source_code/src6/1/conditions.py
# Conditions et opérateurs relationnels

from cs50 import get_int

# Invite l'utilisateur à entrer x
x = get_int("x: ")

# Invite l'utilisateur à entrer y
y = get_int("y: ")

# Comparaison de x et y
if x < y:
    print("x est inférieur à y")
elif x > y:
    print("x est supérieur à y")
else:
    print("x est égal à y")

File: app/cs50x2025/content/french/lectures_source_code/src1/conditions.c
// Conditions et opérateurs relationnels

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Demande à l'utilisateur x
    int x = get_int("x : ");

    // Demande à l'utilisateur y
    int y = get_int("y : ");

    // Compare x et y
    if (x < y)
    {
        printf("x est inférieur à y\n");
    }
    else if (x > y)
    {
        printf("x est supérieur à y\n");
    }
    else
    {
        printf("x est égal à y\n");
    }
}